### ISSUE TITLE
Simple Do not notify UP / 200

### ISSUE THREAD

**Issue by adriangalilea** [louislam/uptime-kuma#4744]
### 📑 I have found these related issues/pull requests

https://github.com/louislam/uptime-kuma/issues/4368
This could work but the solution is complex, this should be implemented on the notification level.

### 🏷️ Feature Request Type

Settings

## New option to all notification "channels" (type in the UI)

> Only send error/down notifications.

And that's it.

## Telegram specific part

> Send Silently
> Sends the message silently. Users will receive a notification with no sound.

So my options are either be annoyed by all successful checks or silent them all and miss important errors.

<img width="1113" alt="image" src="https://github.com/louislam/uptime-kuma/assets/90320947/e2e20ead-c5e7-4714-b0f5-a8ccbcaa618e">

So my Telegram specific suggestion is that it should send all errors non-silently and send me non-errors silently, this way I both know it's working and I don't get annoying notifications while knowing that if an error happens I'll get notified.

### ✔️ Solution

.

**Comment by CommanderStorm** [comment]
Sort of in conflict (or at least interaction is unclear/difficult) with https://github.com/louislam/uptime-kuma/issues/508

I think adding a `disable_notification` toggle [from the telegram api](https://core.telegram.org/bots/api#making-requests) and https://github.com/louislam/uptime-kuma/issues/508 are an alternative.

Maybe a good alternative merging the approaches would be to have this as a mult-selection box (`notify for` => `UP`/`DOWN`/other)

**Comment by adriangalilea** [comment]
> Sort of in conflict (or at least interaction is unclear/difficult) with #508

Completely missed this one, I searched  for notification not alert.

> I think adding a `disable_notification` toggle [from the telegram api](https://core.telegram.org/bots/api#making-requests) and #508 are an alternative.
> 
> Maybe a good alternative merging the approaches would be to have this as a mult-selection box (`notify for` => `UP`/`DOWN`/other). What do you think about these alternatives?

This would be awesome yes.

> Tip
> 
> Notification provider specific changes are really hard to keep up with. If you want to get there faster, you can consider [contributing](https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma) a change to the notification provider[1](#user-content-fn-1-f9a3776cd467acfc55e0bbd034004a5d)[2](#user-content-fn-2-f9a3776cd467acfc55e0bbd034004a5d)

I could totally help with telegram implementation once #508 is done, I've worked a lot with the telegram API and currently working full stack and typescript.

But as of now #508 blocks this.

**Comment by CommanderStorm** [comment]
Just noticed that `disable_notification` is already an option we implement.
=> don't think anything is needed in this department

What #508 needs should be pretty simple to implement.

The shared part of the notification dialog is here:

https://github.com/louislam/uptime-kuma/blob/dbbc79a05a2036176c8b011401c915c0db3384ca/src/components/NotificationDialog.vue

Notifications in the backend are send here

https://github.com/louislam/uptime-kuma/blob/dbbc79a05a2036176c8b011401c915c0db3384ca/server/notification.js#L164-L170

**Comment by bilogic** [comment]
Hi, I thought to check if we are facing the same problem:

I would like to know audibly and visually if 

1. There are any DOWN children (sound A)
2. All children are UP (sound B)

In order to have different sounds, I sent 1 and 2 into different group chats, but that created another issue: I can't easily see if the latest status is UP or DOWN.

After reading earlier comments, I thought it might make sense to have a 3rd chat that is always silent, but has all messages. Another alternative is to have the bot modify the title of the group chat with an icon?

Or is there some way we can meet all requirements using only a single group chat?
